src/App/Fossa/Config/Report.hs

```diff
@@ -52,8 +55,24 @@ instance Show ReportType where
 -- TODO: Add support for text-format reports
 data ReportOutputFormat
   = ReportJson
+  | ReportMarkdown
+  | ReportSPDX
   -- ReportPretty
-  deriving (Eq, Ord, Show)
+  deriving (Eq, Ord, Enum, Bounded)
```

Comment:
Remove `TODO` and `ReportPretty`.  We had originally planned to reproduce the text reports from v1, but with markdown and SPDX, it seems unlikely that we'll go down that route.